pages/conference/_eventType/index.vue

```diff
@@ -4,14 +4,39 @@
             <core-h1 :title="title"></core-h1>
             <p class="intro">{{ intro }}</p>
         </i18n-page-wrapper>
+        <i18n-page-wrapper
+            v-if="eventType === 'talks'"
+            class="px-4 sm:px-8 md:px-16 lg:px-32 pb-6 lg:pb-12"
+            custom-x
+            custom-y
+        >
+            <p class="filter ml-2">Filter</p>
```

Comment:
- Change the class name would be more comprehensible. 
- Move `ml-2` to css block.

```suggestion
            <p class="filterTitle">Filter</p>
```